autobuild/ios/artifacts/autobuild_ios_2_build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh -e
+
+# autobuild_2_build: actual build process
+
+
+####################
+###  PARAMETERS  ###
+####################
+
+
+source "$(dirname "${BASH_SOURCE[0]}")/../../ensure_THIS_JAMULUS_PROJECT_PATH.sh"
+
+###################
+###  PROCEDURE  ###
+###################
+
+cd "${THIS_JAMULUS_PROJECT_PATH}"
+
+qmake -spec macx-xcode Jamulus.pro
```

Comment:
I would have expected the spec to be the iOS build rather than the macx target here?

Reply:
Yes. But that's what builds for iOS

Reply:
Since the build includes the iOS version of Qt, probably `qmake` knows it is building for iOS. Certainly the log shows a line for building `ios/sound.mm`